controllers/telemetry/logpipeline_controller_test.go

```diff
@@ -308,6 +308,21 @@ var _ = Describe("LogPipeline controller", Ordered, func() {
 				return err
 			}, timeout, interval).Should(BeNil())
 		})
+		It("Should have the correct priority class", func() {
```

Comment:
We should test the same for the otel collectors.

Reply:
done